tests/integration/kubernetes/tests_common.sh

```diff
@@ -97,7 +97,12 @@ delete_pod_if_new() {
 exec_host() {
 	node="$1"
 	# `kubectl debug` always returns 0, so we hack it to return the right exit code.
-	command="${@:2}"
+	command="${*:2}"
```

Comment:
If those changes are not directly related to the Policy changes, I would prefer to move them into a separate PR. @GabyCT has been trying to improve this function too, related to https://github.com/kata-containers/kata-containers/issues/10081.

Reply:
Yeah, I was just annoyed by these non-descriptive CI failures and wanted to see if I can do something about it. Happy to remove.